index.d.ts

```diff
@@ -41,6 +41,30 @@ export type Options = {
 		setTimeout: typeof global.setTimeout;
 		clearTimeout: typeof global.clearTimeout;
 	};
+
+	/**
+	You can abort the promise using [`AbortController`](https://developer.mozilla.org/en-US/docs/Web/API/AbortController).
+
+	**Requires Node.js 16 or later.*
+
+	@example
+	```
+	import pTimeout from 'p-timeout';
+
+	const delayedPromise = () => setTimeout(3000);
+
+	const abortController = new AbortController();
+
+	pTimeout(delayedPromise(), 2000, undefined, {
```

Comment:
Missing `await`

Reply:
In this example, when using `await pTimeout`, timeout error will be thrown before abortController.abort is executed.
I think this example needs to show how to abort pTimeout promise.

Reply:
You could either move the settimeout call to be before or use an async-IIFE. Floating promises are bad.